packages/@react-spectrum/test-utils/src/events.ts

```diff
@@ -22,6 +22,21 @@ export function triggerPress(element, opts = {}) {
   fireEvent.click(element, {detail: 1, ...opts});
 }
 
+// Triggers a "touch" event on an element.
+export function triggerTouch(element, opts = {}) {
+  fireEvent.pointerDown(element, {pointerType: 'touch', ...opts});
+  fireEvent.pointerUp(element, {pointerType: 'touch', ...opts});
+}
+
+// Triggers a "longPress" event on an element.
+const DEFAULT_THRESHOLD = 500;
```

Comment:
DEFAULT_LONG_PRESS_TIME maybe? Should this be an export since the test used it?

Reply:
Good idea. Will fix in new commits.

FYI: DEFAULT_THRESHOLD is taken from 'packages/@react-aria/interactions/src/useLongPress.ts' (https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/interactions/src/useLongPress.ts#:~:text=const%20DEFAULT_THRESHOLD%20%3D%20500%3B) but DEFAULT_LONG_PRESS_TIME is more clearer and exporting it makes sense 👍